src/app/zap-templates/zcl/data-model/chip/binding-cluster.xml

```diff
@@ -16,27 +16,23 @@ limitations under the License.
 -->
 <configurator>
   <domain name="CHIP" />
+
+  <struct name="TargetStruct">
+    <cluster code="0x001e"/>
+    <item name="fabricIdx" type="FABRIC_IDX"/>
+    <item name="nodeId" type="NODE_ID" optional="true"/>
+    <item name="groupId" type="GROUP_ID" optional="true"/>
+    <item name="endpointId" type="ENDPOINT_NO" optional="true"/>
+    <item name="clusterId" type="CLUSTER_ID" optional="true"/>
```

Comment:
```suggestion
    <item name="FabricIndex" type="FABRIC_IDX"/>
    <item name="Node" type="NODE_ID" optional="true"/>
    <item name="Group" type="GROUP_ID" optional="true"/>
    <item name="Endpoint" type="ENDPOINT_NO" optional="true"/>
    <item name="Cluster" type="CLUSTER_ID" optional="true"/>
```
That's what the spec has, at least, and I don't see any reason we shouldn't just implement the spec here.